docs/install/index.md

```diff
@@ -1,5 +1,133 @@
-There are a number of different methods to install and run Coder:
+1. Install Coder
+
+   <div class="tabs">
+
+   ## Install Script (Linux/macOS)
+
+   The easiest way to install Coder is to use our
+   [install script](https://github.com/coder/coder/blob/main/install.sh) for
+   Linux and macOS.
+
+   ```shell
+   curl -fsSL https://coder.com/install.sh | sh
+   ```
+
+   You can preview what occurs during the install process:
+
+   ```shell
+   curl -fsSL https://coder.com/install.sh | sh -s -- --dry-run
+   ```
+
+   You can modify the installation process by including flags. Run the help
+   command for reference:
+
+   ```shell
+   curl -fsSL https://coder.com/install.sh | sh -s -- --help
+   ```
+
+   ## Homebrew
+
+   Install Coder from our official
+   [Homebrew tap](https://github.com/coder/homebrew-coder)
+
+   ```shell
+   brew install coder/coder/coder
+   ```
+
+   ## Windows
+
+   Install using
+   [`winget`](https://learn.microsoft.com/en-us/windows/package-manager/winget/#use-winget)
+   package manager
+
+   ```powershell
+   winget install Coder.Coder
+   ```
```

Comment:
We also have a `.msi` installer. Can we mention both under the Windows tab?

Reply:
We do not have a `.msi` but a `.exe` installer, and I have included that in the System packages section. I understood your idea to group all Windows installation methods.

I do not favor grouping based on OS type but with installation method. I want to have an organization similar to [Terrfaorm installation docs](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli#install-terraform) where we can have sub-tabs within a tab.

Reply:
When I started working on this, I did reorganized it on the basis of OS. Let me know what you think.